momento-sdk/src/main/java/momento/sdk/SubscriptionWrapper.java

```diff
@@ -155,15 +157,23 @@ private void handleSubscriptionItem(_SubscriptionItem item) {
 
   private void handleSubscriptionDiscontinuity(_SubscriptionItem discontinuityItem) {
     logger.debug(
```

Comment:
If we're logging here, does it make sense for the onDiscontinuity and onHeartbeat default implementations to log?

Reply:
I don't think so. We should remove logging from the default implementations and use trace logging here, following the [JS SDK conventions](https://github.com/momentohq/client-sdk-javascript/blob/main/packages/client-sdk-nodejs/src/internal/pubsub-client.ts#L248-L264)